plugin/al/lib/AL_USDMaya/AL/usdmaya/fileio/translators/TranslatorContext.h

```diff
@@ -261,8 +261,9 @@ struct TranslatorContext
     /// \brief  this method is used after a variant switch to check to see if the prim types have
     /// changed in the
     ///         stage, and will update the internal state accordingly.
+    /// \param affectedPaths paths that being affected during translation
```

Comment:
Small nit-pick: "that being affected" -> "that are being affected" or more simply "affected". The same typo is present in other modified header files.

Reply:
Should have been fixed in commit 939a550.